instrumentation/opentelemetry-instrumentation-botocore/tests/conftest.py

```diff
@@ -118,4 +118,85 @@ def scrub_response_headers(response):
     This scrubs sensitive response headers. Note they are case-sensitive!
     """
     response["headers"]["Set-Cookie"] = "test_set_cookie"
+    # SDK validates response length which we modify by pretty-printing JSON.
+    # Content length does not affect our instrumentation so just drop it.
+    response["headers"].pop("Content-Length", None)
     return response
+
+
+class LiteralBlockScalar(str):
+    """Formats the string as a literal block scalar, preserving whitespace and
+    without interpreting escape characters"""
+
+
+def literal_block_scalar_presenter(dumper, data):
```

Comment:
This is the standard boilerplate we've used in other codebases with the below tweaks